gp2gp-translator/src/main/java/uk/nhs/adaptors/pss/translator/service/InboundMessageMergingService.java

```diff
@@ -65,6 +65,45 @@ public boolean canMergeCompleteBundle(String conversationId) throws ValidationEx
         return undeletedLogs.stream().allMatch(log -> log.getUploaded().equals(true));
     }
 
+    public void attachmentContainsSkeleton (List<PatientAttachmentLog> attachmentLogs, InboundMessage inboundMessage) throws SAXException, TransformerException {
```

Comment:
This can be a private method. Also the method name should more or less say what it does (but I don't have a great name suggestion, sorry)